library/cc/executor.h

```diff
@@ -1,15 +1,19 @@
 #pragma once
 
-// NOLINT(namespace-envoy)
-
 #include <functional>
 #include <memory>
 
+namespace Envoy {
+namespace Platform {
+
 class Executor {
```

Comment:
I'm still not certain that we need this. If callbacks in this context are `std::function` then they can have all the state embedded they need for dispatch. I'm not sure there's really a need for the new interface, and I'm a little bit hesitant to introduce one that bindings will come to rely on.